scalableminds/webknossos
# Iterate UI for AI job modal

### Steps to test:
- set `isWkorgInstance` and `jobsEnabled` to true in `application.conf`
- open annotation and click "AI analysis" button. click through the modal.
- make sure its working in the same cases as before, such as when viewing a dataset as a superuser
- make sure the "Merge this volume annotation with its fallback layer" in the context menu of a segment layer is still working. 
*NB*: it seemed not to be working before and still isnt, at least for me. its probably important to test this in depth during review :sweat_smile: 

### TODOs:
- [x] revisit description
- [x] exchange `Alert` component
- [x] use ai job enum
- [x] improve layout
- [x] make sure materialize volume annotation is working

### Issues:
- fixes #7268

------
(Please delete unneeded items, merge only when none are left open)
